tests/cupy_tests/random_tests/test_generator.py

```diff
@@ -1090,6 +1090,9 @@ class TestWeibull(RandomGeneratorTestCase):
     def test_weibull(self):
         self.generate(a=self.a, size=(3, 2))
 
+    def test_weibull_size_none(self):
+        self.generate([[0.5, 1.0, 3.0]])
```

Comment:
```suggestion
        self.generate([[0.5, 1.0, 3.0]], size=None)
```

Lets make it explicit :)